src/sentry/static/sentry/app/components/versionHoverCard.tsx

```diff
@@ -73,7 +73,7 @@ class VersionHoverCard extends React.Component<Props, State> {
 
   getBody() {
     const {releaseVersion, release, deploys} = this.props;
-    if (release === undefined || deploys === undefined) {
+    if (release === undefined || !deploys) {
```

Comment:
We also have handy utility for this:
https://github.com/getsentry/sentry/blob/b8bf85948d4c4832c60ab31e9f1a5a3c39fd53cf/src%2Fsentry%2Fstatic%2Fsentry%2Fapp%2Futils.tsx#L115-L117

Reply:
oooh sweet let me switch to that